spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -634,27 +682,57 @@ po/skipper-2408247821-50z31       1/1       Running   0          3m
 ...
 ...
 ----
-
+====
 
 [[getting-started-kubernetes-create-stream]]
-==== Create Streams
-
-This section describes how to create streams (using Skipper). To do so:
+==== Creating Streams
+
+This section describes how to create streams (by using Skipper). The following sections describe how to do so:
```

Comment:
I like this listing!

Reply:
It's a technique called information mapping. It helps to make sense of complex content (in this case, instructions in which each step is its own set of instructions).